tests/conftest.py

```diff
@@ -2210,13 +2211,19 @@ def fixture_malformed_hermes_deliverables(hermes_deliverables: dict) -> dict:
 
 @pytest.fixture(name="rnafusion_multiqc_json_metrics")
 def fixture_rnafusion_multiqc_json_metrics(rnafusion_analysis_dir) -> dict:
-    """Returns a the content of a mock multiqc json file."""
+    """Returns the content of a mock Multiqc JSON file."""
     return read_json(file_path=Path(rnafusion_analysis_dir, "multiqc_data.json"))
 
 
+@pytest.fixture(name="tower_id")
+def fixture_tower_id() -> int:
```

Comment:
What is the difference between the `tower_id` and `tower_run_id`. If none I think we should only use one of them in the code.

Reply:
good catch! there's no difference between them